maintenance/monitor/monitor-component-visual.md

```diff
@@ -9,7 +9,9 @@ Use Grafana dashboard to view {{ site.prodname }} component metrics.
 
 ### Value
 
-Using Grafana can be beneficial by providing a means to visualize metrics through graphs that can help you quickly identify unusual activity.
+Using Grafana can be beneficial by providing a means to visualize metrics through graphs that can help you quickly identify unusual activity, following image shows some of the graphs and metrics that are available for you to leverage in order to achieve this goal.
```

Comment:
I guess I'd put a period in between "activity" and "[The] following image." I'll approve since this is a cherry-pick though - presumably someone must have thought a run-on sentence was okay.

Reply:
That's a good spot and not sure why I didn't spot that one myself.  I'll do a quick PR to fix that in master.